recipe/gen_patch_json.py

```diff
@@ -1488,6 +1488,11 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             if record["version"] in ["0.3.4", "0.3.6", "0.3.9", "0.4.0"]:
                 _replace_pin("dask >=2.19.0,!=2021.3.0", f"dask =={dask_sql_map[record['version']]}", deps, record)
 
+            # make dask/uvicorn pinnings consistent for older builds of 2022.10.1
```

Comment:
Could we link a thread, issue, or PR here? Something that gives the reader context about what is changing and why

Reply:
Yup, will link your comment from the original release PR:

https://github.com/conda-forge/dask-sql-feedstock/pull/46#issuecomment-1291416642

Reply:
Thanks Charles! 🙏